src/main/java/liquibase/nosql/database/AbstractNoSqlDatabase.java

```diff
@@ -0,0 +1,329 @@
+        return null;
+    }
+
+    @Override
+    public String escapeViewName(final String catalogName, final String schemaName, final String viewName) {
+        return null;
+    }
+
+    @Override
+    public String escapeStringForDatabase(final String string) {
+        return null;
+    }
+
+    @Override
+    public abstract void dropDatabaseObjects(final CatalogAndSchema schemaToDrop) throws LiquibaseException;
+
+    @Override
+    public boolean supportsRestrictForeignKeys() {
+        return false;
+    }
+
+    @Override
+    public String escapeConstraintName(final String constraintName) {
+        return null;
+    }
+
+    @Override
+    public List<DatabaseFunction> getDateFunctions() {
+        // irrelevant (will never be called as this is not SQL being processed)
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public boolean supportsForeignKeyDisable() {
+        return false;
+    }
+
+    @Override
+    public boolean disableForeignKeyChecks() {
+        return false;
+    }
+
+    @Override
+    public void enableForeignKeyChecks() {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public boolean isCaseSensitive() {
+        return true;
+    }
+
+    @Override
+    public boolean isReservedWord(final String string) {
+        return false;
+    }
+
+    @Override
+    public boolean isFunction(String string) {
+        return false;
+    }
+
+    @Override
+    public int getDataTypeMaxParameters(String dataTypeName) {
+        return 0;
+    }
+
+    @Override
+    public boolean dataTypeIsNotModifiable(String typeName) {
+        return false;
+    }
+
+    @Override
+    public String generateDatabaseFunctionValue(DatabaseFunction databaseFunction) {
+        return null;
+    }
+
+    @Override
+    public boolean createsIndexesForForeignKeys() {
+        //Not applicable
+        return false;
+    }
+
+    @Override
+    public boolean supportsPrimaryKeyNames() {
+        //Not applicable
+        return false;
+    }
+
+    @Override
+    public boolean supportsNotNullConstraintNames() {
+        //Not applicable
+        return false;
+    }
+
+    @Override
+    public boolean supportsBatchUpdates() {
+        return false;
+    }
+
+    @Override
+    public boolean requiresExplicitNullForColumns() {
+        //Not applicable
+        return false;
+    }
+
+    @Override
+    public String getSystemSchema() {
+        return null;
+    }
+
+    @Override
+    public String escapeDataTypeName(final String dataTypeName) {
+        return null;
+    }
+
+    @Override
+    public String unescapeDataTypeName(final String dataTypeName) {
+        return null;
+    }
+
+    @Override
+    public String unescapeDataTypeString(final String dataTypeString) {
+        return null;
+    }
+
+    @Override
+    public ValidationErrors validate() {
+        return null;
```

Comment:
@filipelautert: what's this supposed to do / does this need to be implemented?
if so: how can this be tested?

Reply:
Seems it is not even used... seems some old method that was ignored. Please ignoreit.